src/fe-text/mainwindows.h

```diff
@@ -5,7 +5,7 @@
 #include "term.h"
 
 #define WINDOW_MIN_SIZE 2
-#define NEW_WINDOW_WIDTH 10
+#define NEW_WINDOW_WIDTH 20 /* must be >= MIN_SCREEN_WIDTH defined in term.c */
```

Comment:
What about using `G_STATIC_ASSERT` [1] to enforce this invariant?

[1] https://developer.gnome.org/glib/stable/glib-Miscellaneous-Macros.html#G-STATIC-ASSERT:CAPS

Reply:
nice idea but won't work unless we make MIN_SCREEN_WIDTH public